### ISSUE TITLE
Moving Cards from Seat does not work as in PCIO

### ISSUE THREAD

**Issue by goldenurg** [ArnoldSmith86/virtualtabletop#1181]
https://virtualtabletop.io/0cn8

Sit in seat.
Drag card your hand.
Use the undeal.

In PCIO this moves the card to holder.
In VTT this moves the various pieces of the seat to holder.

**Comment by 96LawDawg** [comment]
Should be fixed by #933.  However, I was unable to test if it worked because the switch from the old style pr-test rooms to the new server.  I could not figure out how to trigger a new-style pr-test room to open in 933.

**Comment by goldenurg** [comment]
Would it help to have the original pcio file?
[bugreport.pcio.gz](https://github.com/ArnoldSmith86/virtualtabletop/files/8401606/bugreport.pcio.gz)

**Comment by ArnoldSmith86** [comment]
> However, I was unable to test if it worked because the switch from the old style pr-test rooms to the new server. I could not figure out how to trigger a new-style pr-test room to open in 933.

I updated the link in the first comment. The `test.virtualtabletop.io` addresses only work for PRs >=1087. I'll have a look at the script someday soon so we can hopefully avoid breaking (and updating) all the links.

**Comment by ArnoldSmith86** [comment]
> I'll have a look at the script someday soon so we can hopefully avoid breaking (and updating) all the links.

The links for older PRs should no longer break.

**Comment by 96LawDawg** [comment]
Tested and #933 does fix this.

**Comment by RaphaelAlvez** [comment]
@96LawDawg should this be closed?

**Comment by 96LawDawg** [comment]
No.  Why?  This is still broken after #1116 because this deals with `MOVE` **from** a seat.

**Comment by RaphaelAlvez** [comment]
but you stated that it is fixed by #933
